### PR TITLE
Changed Chart to allow onClick to work for bar and point types

### PR DESCRIPTION
**Pull request by ericsoderberghp**
#### What does this PR do?

Changed Chart to allow onClick to work for bar and point types

#### Where should the reviewer start?

Chart.js

#### What testing has been done on this PR?

grommet designer

#### How should this be manually tested?

hmm???

#### Do the grommet docs need to be updated?

automatic

#### Should this PR be mentioned in the release notes?

yes

#### Is this change backwards compatible or is it a breaking change?

backwards compatible

### CODE REVIEW

File: src/js/components/Chart/doc.js [hunk 1]
```diff
@@ -58,8 +58,8 @@ export const doc = Chart => {
     id: PropTypes.string.description(`A unique identifier for the Chart. This
       is required if more than one Chart is shown and they use color
       gradients.`),
-    onClick: PropTypes.func.description(`Called when the user clicks on it.
-      This is only available when the type is line or area.`),
+    onClick: PropTypes.func.description(`Called when the user clicks on the
+     visualization. values[].onClick should be used for types bar and point.`),
```
Comment (ShimiSun) [inline src/js/components/Chart/doc.js]: what happens if I use it for line chart?

**Comment by ericsoderberghp** [comment]
> The code looks good.
> Since we are allowing `<path>` to be an interactive element now, I would like to see how it performs and whether it is compliant with accessibility standards of interactive elements.

We already had `onClick` on path elements for line and area types, this is just adding it for bar and point types.

File: src/js/components/Chart/doc.js (hunk 1 above)
Comment (ericsoderberghp) [reply src/js/components/Chart/doc.js]: Clicking on the line will trigger the function.

Comment (ShimiSun) [reply src/js/components/Chart/doc.js]: So why does it say it should be used for 
>values[].onClick should be used for types bar and point

...and not for a line chart? I feel like I'm missing something.

Comment (ericsoderberghp) [reply src/js/components/Chart/doc.js]: I've revised the documentation for `onClick` in an attempt to make it clearer.

**Pull request merged**